test/TestUtils.hs

```diff
@@ -122,3 +126,16 @@ isPkgDbAt _ _ = False
 
 instance IsString ModuleName where
     fromString = mkModuleName
+
+-- | Bracketing function for test/data/options-cradle environment to install tagged in package-db-a
+bracketTagged :: IO a -> IO a
+bracketTagged m = do
+    withDirectory_ "test/data/options-cradle" $ do
```

Comment:
Hiding the directory in here is confusing, just took me more than 30sec to find that :p